README.md

```diff
@@ -9,46 +9,66 @@
 
 </div>
 
+## Table of Contents
+
+-   [Overview](#overview)
+-   [Usage](#usage)
+    -   [General](#general)
+-   [Authorization](#authorization)
+-   [Setting the build name and build number when publishing build-info to Artifactory](#setting-the-build-name-and-build-number-when-publishing-build-info-to-artifactory)
+-   [Setting JFrog CLI version](#setting-jfrog-cli-version)
+-   [Downloading JFrog CLI from Artifactory](#downloading-jfrog-cli-from-artifactory)
+-   [Set up a FREE JFrog Environment in the Cloud](#set-up-a-free-jfrog-environment-in-the-cloud)
+-   [Example projects](#example-projects)
+-   [Contributions](#contributions)
+-   [References](#references)
+
+## Overview
+
 This GitHub Action downloads, installs and configures JFrog CLI, so that it can be used as part of the workflow.
```

Comment:
Let's make "JFrog CLI" a link to - 
https://docs.jfrog-applications.jfrog.io/jfrog-applications/jfrog-cli